Replicator/Pusher.cc

```diff
@@ -227,10 +227,7 @@ namespace litecore { namespace repl {
             req[kConflictIncludesRevProperty] = "true"_sl;
         }
 
-        if (collectionIndex() != kNotCollectionIndex) {
-            req[kCollectionProperty] = collectionIndex();
-        }
-
+        req[kCollectionProperty] = collectionIndex();
```

Comment:
I'm wondering if this will confuse SG for the case of replicating only default collection which uses getRemoteCheckpoint() instead of getCollections().

Reply:
That was one of the considerations, but I believe it will simply be ignored instead.  I asked for confirmation on this.